### PR TITLE
SW-5159 Created list contacts page

### PR DESCRIPTION
**Pull request by josephsmith0705**
Created page to view current deputy contacts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,19 +48,15 @@ Alternatively to set it up not using Docker use below. This hosts it on `localho
...
-`yarn cypress-for-m1-up` <br>
-`yarn cypress-headless` This command will run all the cypress tests <br>
-`yarn cypress-headless --spec "cypress/e2e/A_FILE_NAME.spec.js` will only run the tests in that file
+`docker-compose -f docker/docker-compose.ci.yml up -d --build `
 
-Only thing to note is that if there are any changers locally to a gotmpl file you will need to run<br>
-`cypress-build-down`<br>
-Then the above commands to pull in the latest code to test against.<br>
-You will not have to re-build anything tho if you change the code in a spec file only.
+`yarn cypress-m1 `
```
Comment (kate-49) [inline README.md:58]: Just checking you meant to remove this guidance?

File: cypress/e2e/contacts.cy.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                .children()
+                .each(($el, index) => {
+                    cy.wrap($el).should("contain", expected[index]);
+            });
+        });
+
+        it("should display contact data", () => {
+            cy.get(':nth-child(1) > :nth-child(1) > .name').contains("Minimal Contact");
+            cy.get(':nth-child(1) > :nth-child(2) > .email > a').contains("email@test.com");
+            cy.get(':nth-child(1) > :nth-child(2) > .phone-number').contains("0123456789");
```
Comment (kate-49) [inline cypress/e2e/contacts.cy.js:90]: Just wondering if there's a cleaner way to get them, e.g. would `:nth-child(1) > .name` bring back that contact?

File: cypress/e2e/navigation.cy.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ describe("Navigation bar", () => {
 
     const expected = [
         ["Deputy details", "/supervision/deputies/1"],
+        // ["Contacts", "/supervision/deputies/1/contacts"],
```
Comment (kate-49) [inline cypress/e2e/navigation.cy.js:10]: Feels like we might not want this commented out?

File: internal/server/add_contact_test.go [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+			"isEmpty": "Select whether this contact is the named deputy",
+		},
+	}
+
+	assert.Equal(addContactVars{
+		Path:   "/133",
+		Errors: expectedValidationErrors,
+	}, template.lastVars)
+
+	assert.Nil(returnedError)
```
Comment (kate-49) [inline internal/server/add_contact_test.go:131]: This is a cool test

File: internal/server/add_contact_test.go [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+		"email": {
+			"stringLengthTooLong": "The email must be 255 characters or fewer",
+		},
+		"phoneNumber": {
+			"stringLengthTooLong": "The telephone number must be 255 characters or fewer",
+		},
+		"otherPhoneNumber": {
+			"stringLengthTooLong": "The other telephone number must be 255 characters or fewer",
+		},
+		"contactNotes": {
```
Comment (kate-49) [inline internal/server/add_contact_test.go:192]: Would this test work just as well with only one error? We're just checking a Sirius error can be returned right?

File: web/template/layout/navigation.gotmpl [hunk 6]
```diff
@@ -13,6 +13,16 @@
                             >Deputy details</a
                         >
                     </li>
+                    {{/*<li class="moj-sub-navigation__item">
+                        <a
```
Comment (kate-49) [inline web/template/layout/navigation.gotmpl:17]: Just checking that this should be commented out?

**Review by Nicholassully — APPROVED** [review]

**Pull request merged**